src/ensembl/src/content/app/species-selector/state/speciesSelectorEpics.ts

```diff
@@ -0,0 +1,41 @@
+import { Epic } from 'redux-observable';
+import { of } from 'rxjs';
+import {
+  map,
+  switchMap,
+  delay,
+  filter,
+  distinctUntilChanged
+} from 'rxjs/operators';
+import { isActionOf, ActionType } from 'typesafe-actions';
+
+import * as speciesSelectorActions from 'src/content/app/species-selector/state/speciesSelectorActions';
+
+import humanMockSearchResults from 'tests/data/species-selector/human-search';
+import mouseMockSearchResults from 'tests/data/species-selector/mouse-search';
+
+import { RootState } from 'src/store';
+
+type Action = ActionType<typeof speciesSelectorActions>;
+
+// FIXME: this is a mock implementation of the fetcher
+export const fetchSpeciesSearchResultsEpic: Epic<Action, Action, RootState> = (
+  action$
+) =>
+  action$.pipe(
+    filter(
+      isActionOf(speciesSelectorActions.fetchSpeciesSearchResults.request)
+    ),
+    distinctUntilChanged(
+      (action1, action2) => action1.payload === action2.payload
+    ),
+    switchMap((action) => of(action).pipe(delay(600))),
+    map((action) =>
+      speciesSelectorActions.fetchSpeciesSearchResults.success({
+        text: action.payload,
+        results: action.payload.startsWith('h')
```

Comment:
There can be more species that begin with `h` so can this be `homo` instead just to make it future-proof?

Reply:
This is just a silly mock to choose between two examples of hardcoded data. It will go away entirely when we have real endpoints. I don't think it really matters how silly it is, because it'll be deleted anyway 😄